### ISSUE TITLE
FormBuilderChipsInput giving "The method 'findRenderObject' was called on null" error.

### ISSUE THREAD

**Issue by danielweil** [danvick/flutter_form_builder#572]
Hey, 

I have a couple problems with the FormBuilderChipsInput widget. I use it as a list with multiple options, and when I apply focus on it, the suggestions don't immediately show, only after I type the first letter in the query. (Would love to be able to).

After I select an option, the keyboard vanishes. It would be awesome if there was a parameter that allowed the keyboard to continue opened. After I apply focus again to the field, It gives me the  "The method 'findRenderObject' was called on null" error.

After this, if I leave the page and come back, it gives me the focusNode error.

I want to use this widget in my final project version, but I need this errors fixed. Can you guys give a look at it?

Here are images for the errors.

![image](https://user-images.githubusercontent.com/44274213/99319085-b3d80580-2847-11eb-92cb-5c95e0d601dc.png)
![image](https://user-images.githubusercontent.com/44274213/99319143-cb16f300-2847-11eb-983b-8a0155bb1860.png)

Thanks for all your help!

'

**Comment by danielweil** [comment]
@awhitford @danvick Any update on this issue? This is the only big issue I have opened, as the other ones I was able to hack through.

**Comment by awhitford** [comment]
This might have been fixed with: https://github.com/danvick/flutter_chips_input/commit/b907c64f9f7023cbf8f8192241f88dd2e6dab29e

However, this code has not been released.

**Comment by danielweil** [comment]
Any release date yet?

**Comment by awhitford** [comment]
[flutter_chips_input 1.9.5](https://pub.dev/packages/flutter_chips_input/changelog) was released.  Does that solve your issue?

**Comment by danielweil** [comment]
Have a question, doubt, I don't have flutter_chips_input as a direct dependency. Only flutter_form_builder. I am able to use this new version without having a new Form Builder release? Or do I have to use dev branch?

**Comment by danielweil** [comment]
It worked. 
On a second note, do you guys see possible any of these 2 features?

> I have a couple problems with the FormBuilderChipsInput widget. I use it as a list with multiple options, and when I apply focus on it, the suggestions don't immediately show, only after I type the first letter in the query. (Would love to be able to).

> After I select an option, the keyboard vanishes. It would be awesome if there was a parameter that allowed the keyboard to continue opened. 

These are quoted in this issue, as well. https://github.com/danvick/flutter_form_builder/issues/510

**Comment by awhitford** [comment]
@danielweil I think those are feature requests for the underlying `flutter_chips_input`.  There is still more work to do.  However, stability (no crashes) is more important than new features.

**Comment by danielweil** [comment]
We can work on a merge request. How should we proceed? Any recommendation?

**Comment by awhitford** [comment]
> We can work on a merge request. How should we proceed? Any recommendation?

PR's are welcome.  As I mentioned, start with adding features to the [flutter_chips_input](https://github.com/danvick/flutter_chips_input) control since this project is just a thin wrapper for that functionality.

**Comment by danielweil** [comment]
@danvick @awhitford can you guys have a look at this PR? https://github.com/danvick/flutter_chips_input/pull/73

There was an issue with the CI, but we want to hear some feedback before using more of our time to fix it.

We also has another PR ready, but will wait for feedback on this one as well

**Comment by danielweil** [comment]
@awhitford any updates?

**Comment by deandreamatias** [comment]
This still a issue?
